packages/api/tests/controllers/grantCycle.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require('supertest');
+const app = require('../../server');
+const db = require('../../models');
+
+// TODO make test to make sure relations work with Nomination as expected
+
+describe('POST Grant Cycle Endpoint', () => {
+  let body = {};
+
+  beforeAll(() => {
```

Comment:
What do you think about just moving the `body` definition up out of the `beforeAll` block, since it's just a variable definition (not actually doing any database manipulation or anything).

Reply:
good question. More than move it out of the function, I thought to update it to beforeEach. this way, body resets as a valid request for each test.
second test alters body.openedOn to make the request invalid
third test deletes the name key (with this version it was returning a 400 because of the invalid dates altered in the second test)